### PR TITLE
Add cloudwatch-logs submodule

### PR DESCRIPTION
**Pull request by HuwCampbell**

### CODE REVIEW

**Comment by markhibberd** [comment]
:+1: shallow, @nhibberd is there anything else that needs to be done, builds etc...?

File: mismi-cloudwatch-logs/mafia [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#!/bin/sh -eu
```
Comment (nhibberd) [inline mismi-cloudwatch-logs/mafia]: This can be a symlink `../framework/mafia`

**Comment by nhibberd** [comment]
For the main `mismi-cloudwatch` and `mismi-cloudwatch.cabal` build we just use the normal `job` template, however for the `mismi-cloudwatch.branches` and `mismi-cloudwatch.cabal.branches` we use the `branches-watch` template with region set to 
```
mismi-core/.*
mismi-<project>/.*
framework/.*
```

**Comment by nhibberd** [comment]
:zap: after symlink and builds

**Pull request merged**